This file is part of MXE.
See index.html for further information.

Contains ad hoc patches for cross building.

From 01e3cf1fae96edc0b5e8ccd0bf2c2ea11d0a6987 Mon Sep 17 00:00:00 2001
From: MXE
Date: Sun, 4 Aug 2013 23:32:15 +0200
Subject: [PATCH] comment out production of some file for make install to
 succeed


diff --git a/glade/Makefile.am b/glade/Makefile.am
index f6271f7..fae55bc 100644
--- a/glade/Makefile.am
+++ b/glade/Makefile.am
@@ -13,10 +13,10 @@ if OS_WIN32
 export_symbols = -export-symbols glade.def
 
 install-libtool-import-lib:
-	$(INSTALL) .libs/libglade-2.0.dll.a $(DESTDIR)$(libdir)
+#	$(INSTALL) .libs/libglade-2.0.dll.a $(DESTDIR)$(libdir)
 
 uninstall-libtool-import-lib:
-	-rm $(DESTDIR)$(libdir)/libglade-2.0.dll.a
+#	-rm $(DESTDIR)$(libdir)/libglade-2.0.dll.a
 else
 
 export_symbols = -export-symbols-regex "^[^_].*"
-- 
1.8.1.2

